packages/storybook-builder-vite/codegen-importfn-script.js

```diff
@@ -0,0 +1,48 @@
+const { normalizeStories } = require('@storybook/core-common');
+
+function toImportFnPart(specifier, storyImports) {
+    const { directory, importPathMatcher } = specifier;
+
+    return `
+      async (path) => {
+        if (!${importPathMatcher}.exec(path)) {
+          return;
+        }
+        
+        const pathRemainder = path.substring(${directory.length + 1});
+        const module = (await import('${storyImports}')).default;
+        return module[path]();
```

Comment:
@joshwooding can you tell me a bit about this snippet? Does `storyImports` contain a straight hash of filename => import functions?